lib/tasks/prune_reports.rake

```diff
@@ -43,6 +64,26 @@ UNITS:
       errors << "You must specify the unit of time, .e.g.: unit={#{known_units}}" \
```

Comment:
This trailing \ appears to be a mistake, please remove in your PR to help clean up the existing code.